framework/src/media/MediaWorker.cpp

```diff
@@ -40,13 +40,13 @@ void MediaWorker::startWorker()
 		pthread_attr_t attr;
 		pthread_attr_init(&attr);
 		pthread_attr_setstacksize(&attr, mStacksize);
-		mIsRunning = true;
 		ret = pthread_create(&mWorkerThread, &attr, static_cast<pthread_startroutine_t>(MediaWorker::mediaLooper), this);
 		if (ret != OK) {
 			medvdbg("Fail to create worker thread, return value : %d\n", ret);
 			--mRefCnt;
 			return;
 		}
+		mIsRunning = true;
```

Comment:
This line should be removed. And "mIsRunning = false" should be inserted inside of the "if" failure case.

Reply:
You're right. I will  fix.

Reply:
If mRefCnt is 1, it is always set to true.
So, you can set it false or not.